isisd/isis_cli.c

```diff
@@ -46,16 +46,21 @@
 /*
  * XPath: /frr-isisd:isis/instance
  */
-DEFPY_YANG_NOSH(router_isis, router_isis_cmd, "router isis WORD$tag",
-	   ROUTER_STR
-	   "ISO IS-IS\n"
-	   "ISO Routing area tag\n")
+DEFPY_YANG_NOSH(router_isis, router_isis_cmd,
+		"router isis [{WORD$tag | vrf NAME$vrf_name}]",
```

Comment:
Shouldn't this be `"router isis WORD$tag [vrf NAME$vrf_name]"`? Else the tag is optional and you need it for the yang key. Same goes for the other commands where the vrf was added.

Reply:
Thanks for your review.
By mistake the WORD$tag went into optional, now it resolved and commited.